datasets/coached_conv_pref/coached_conv_pref.py

```diff
@@ -0,0 +1,175 @@
+                                    "startIndex": datasets.Value("int32"),
+                                    "endIndex": datasets.Value("int32"),
+                                    "text": datasets.Value("string"),
+                                    "annotations": datasets.Sequence(
+                                        {
+                                            "annotationType": datasets.features.ClassLabel(
+                                                names=[
+                                                    "ENTITY_NAME",
+                                                    "ENTITY_PREFERENCE",
+                                                    "ENTITY_DESCRIPTION",
+                                                    "ENTITY_OTHER",
+                                                    "",
+                                                ]
+                                            ),
+                                            "entityType": datasets.features.ClassLabel(
+                                                names=[
+                                                    "MOVIE_GENRE_OR_CATEGORY",
+                                                    "MOVIE_OR_SERIES",
+                                                    "PERSON",
+                                                    "SOMETHING_ELSE",
+                                                    "",
+                                                ]
+                                            ),
+                                        }
+                                    ),
+                                }
+                            ),
+                        }
+                    ),
+                }
+            ),
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+
+        data_dir = dl_manager.download_and_extract(_URLs)
+
+        # Dataset is a single corpus (does not contain any split)
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir["dataset"]),
+                    "split": "train",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath, split):
+        """ Yields examples. """
+
+        # Empty Segment list with annotations dictionary
+        # First prompt of a conversation does not contain the segment dictionary
+        # We are setting it to None values
+        segments_empty = [
+            {
+                "startIndex": 0,
+                "endIndex": 0,
+                "text": "",
+                "annotations": [
+                    {
+                        "annotationType": "",
+                        "entityType": "",
+                    }
+                ],
```

Comment:
```suggestion
                "annotations": [],
```

you can simply add no annotations in this case no ?

Reply:
Yes, indeed. This looks much cleaner!